File: api.unitethiscity.com/Models/StatTip.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace com.unitethiscity.api.Models
{
    public class StatTip
    {
        public int Id { get; set; }
        public int BusId { get; set; }
        public Guid BusGuid { get; set; }
        public string Name { get; set; }
        public DateTime Timestamp { get; set; }
        public string TimestampAsString { get; set; }
        public string TimestampSortable { get; set; }
        public string TipText { get; set; }
        public string LocationName { get; set; }
    }
}

File: android/unitethiscity/uniteThisCity/src/main/java/com/unitethiscity/data/StatTip.java
package com.unitethiscity.data;

/**
 * Created by Don on 2/16/2016.
 */
public class StatTip {

    int ID;
    int BusinessID;
    String BusinessGUID;
    String Name;
    String Timestamp;
    String TimestampAsString;
    String TimestampSortable;
    String TipText;
    String LocationName;

    public StatTip(int id, int businessID, String businessGUID, String name, String timestamp, String timestampAsString, String timestampSortable, String tipText, String locationName) {
        ID = id;
        BusinessID = businessID;
        BusinessGUID = businessGUID;
        Name = name;
        Timestamp = timestamp;
        TimestampAsString = timestampAsString;
        TimestampSortable = timestampSortable;
        TipText = tipText;
        LocationName = locationName;
    }

    public int getID() {
        return ID;
    }

    public int getBusinessID() {
        return BusinessID;
    }

    public String getBusinessGUID() {
        return BusinessGUID;
    }

    public String getName() {
        return Name;
    }

    public String getTimestamp() {
        return Timestamp;
    }

    public String getTimestampAsString() {
        return TimestampAsString;
    }

    public String getTimestampSortable() {
        return TimestampSortable;
    }

    public String getTipText() {
        return TipText;
    }

    public String getLocationName() {
        return LocationName;
    }

}
